test/util/ArrayLoggerWriter.hx

```diff
@@ -45,4 +45,10 @@ class ArrayLoggerWriter extends Base implements ILoggerWriter {
     private var file: sys.io.FileOutput;
     private var lines: Array<String> = new Array<String>();
 
+    public function copy(): ArrayLoggerWriter{
+        var newCopy: ArrayLoggerWriter =  new ArrayLoggerWriter();
```

Comment:
This could be final instead of var, and type can be omitted since it is inferred by the expression.